src/components/DesignTheme/View/EventoView.jsx

```diff
@@ -0,0 +1,317 @@
+              /(<([^>]+)>)/g,
+              '',
+            ) ? (
+              <>
+                <h4>Contatti</h4>
+                <div className="card card-teaser rounded shadow mt-3">
+                  <Icon icon={'it-telephone'} />
+                  <div className="card-body pr-3">
+                    <p className="card-text">
+                      {content?.organizzato_da_esterno?.data.replace(
+                        /(<([^>]+)>)/g,
+                        '',
+                      )}
+                    </p>
+                  </div>
+                </div>
+                <h5 className="mt-4">Con il supporto di:</h5>
+                {content?.evento_supportato_da?.map(item => (
+                  <OfficeCard
+                    key={item['@id']}
+                    office={item}
+                    extended={true}
+                    icon={'it-pa'}
+                  />
+                ))}
+              </>
+            ) : null}
+            {content?.organizzato_da_interno?.length > 0 ? (
+              <>
+                {content?.evento_organizzato_da_interno?.map(item => (
+                  <OfficeCard
+                    key={item['@id']}
+                    office={item}
+                    extended={true}
+                    icon={'it-telephone'}
+                  />
+                ))}
+                <h5 className="mt-4">Con il supporto di:</h5>
+                {content?.evento_supportato_da?.map(item => (
+                  <OfficeCard
+                    key={item['@id']}
+                    office={item}
+                    extended={true}
+                    icon={'it-pa'}
+                  />
+                ))}
+              </>
+            ) : null}
+            {content && (
+              <Events content={content} show_image={true} title={null} />
+            )}
+            {content?.ulteriori_informazioni?.data.replace(
+              /(<([^>]+)>)/g,
+              '',
+            ) && (
+              <RichTextArticle
+                content={content?.ulteriori_informazioni?.data}
+                tag_id="additional-info"
+                title={'Ulteriori informazioni'}
+              />
+            )}
+            {content?.patrocinato_da && (
+              <RichTextArticle
+                content={content?.patrocinato_da}
+                tag_id="patrocinio"
+                title={'Patrocinato da:'}
+              />
+            )}
+            {content?.sponsor?.data.replace(/(<([^>]+)>)/g, '') && (
+              <RichTextArticle
+                content={content?.sponsor?.data}
+                tag_id="sponsor"
+                title={'Sponsor:'}
+              />
+            )}
+            {content?.box_aiuto?.data.replace(/(<([^>]+)>)/g, '') && (
+              <HelpBox
+                text={content?.box_aiuto?.data.replace(/(<([^>]+)>)/g, '')}
+              />
+            )}
+            {content?.relatedItems?.length > 0 ? (
```

Comment:
Pensavi di fare le news con questi correlati? Perché in realtà le novità si potrebbero mostrare automaticamente se c'è una relazione fra notizia e eventi... Ne è stato parlato? non è detto che questa soluzione non vada bene, ma abbiamo soluzioni diverse in giro.

Reply:
No, non ne abbiamo parlato, ho preso esempio dalle altre view che hanno correlati